### PR TITLE
Support multiple pegmen (part 2)

### PR DESCRIPTION
**Pull request by molly-moen**
Part 2 of the pre-work for supporting multiple pegmen in Maze.

See [Part 1](https://github.com/code-dot-org/maze/pull/41)and the spec linked below for more context.

This PR includes changes to no longer hard code all pegman html elements with ids such as `pegman` and `movePegman`. If the maze only has one pegman those ids will stay the same (to avoid breaking things for code-dot-org use of Maze, as it occasionally references those ids directly), but for multiple-pegman mazes the ids will be `pegman-${id1}`, `pegman-${id2}`, etc.

This change involved moving the generation of these ids from hard coded strings throughout the repo into a helper util method that takes the prefix and the pegman id. I also added constants for the various per-pegman elements.

## Testing
Did lots of manual testing on this (tested all subtypes, failure, success conditions) as well as running the automated tests from both Maze and `code-dot-org`. 

## Links
- [Spec](https://docs.google.com/document/d/12jAhyv-2TuRGdQA97hIqEBV8ghqazhl6NqxmVrArYzk/edit#heading=h.vbssf3j9joeg)
- [Jira](https://codedotorg.atlassian.net/browse/CSA-297)

### CODE REVIEW

File: src/drawMap.js [hunk 1]
```diff
@@ -26,6 +26,43 @@ function displayPegman(skin, pegmanIcon, clipRect, x, y, frame, squareSize = 50)
   clipRect.setAttribute('y', pegmanIcon.getAttribute('y'));
 }
 
+function addNewPegman(skin, pegmanId, x, y, direction, svg) {
```
Comment (molly-moen) [inline src/drawMap.js:29]: this method is almost entirely a copy/paste from below. I moved it so we can call it in the future for adding new pegman to a maze on the fly.

Comment (jmkulwik) [reply src/drawMap.js:29]: Diff in case any else is curious
image

File: src/utils.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+const DEFAULT_PEGMAN_ID = require('./constants').DEFAULT_PEGMAN_ID;
```
Comment (jmkulwik) [inline src/utils.js:1]: NIT: Is Maze using ES6? If so, you should be able to use
`import {DEFAULT_PEGMAN_ID} from './constants'`

**Review by jmkulwik — APPROVED** [review]
Nice change! This looks great!

File: src/utils.js (hunk 2 above)
Comment (molly-moen) [reply src/utils.js:1]: yep will update! That syntax does work in Maze but isn't used consistently.

File: src/drawMap.js (hunk 1 above)
Comment (molly-moen) [reply src/drawMap.js:29]: thanks!

**Pull request merged**